doc/source/conf.py

```diff
@@ -3,6 +3,10 @@
 # SimPhoNy-Mayavi documentation build configuration file
 #
 import sys
+import os
+
+sys.path.insert(
+    0, os.path.abspath(os.path.join(os.path.dirname(__name__), '..', '..')))
```

Comment:
Even when setup develop, it still can't find it. Do you have a PYTHONPATH defined by any chance?